lib/utils.pm

```diff
@@ -754,6 +754,11 @@ sub power_action {
             record_soft_failure('bsc#1055462');
             $shutdown_timeout *= 3;
         }
+        if (get_var("OFW") && check_var('DISTRI', 'opensuse') && check_var('TEST', 'create_hdd_gnome')) {
```

Comment:
Please, try to avoid checking TEST variable value, as test can be renamed. Use some other variables to identify the test suite which have low chance of being renamed, e.g. DESKTOP and PUBLISH_HDD